ortools/math_opt/solvers/xpress_solver_test.cc

```diff
@@ -59,16 +68,18 @@ INSTANTIATE_TEST_SUITE_P(XpressLpModelSolveParametersTest,
 
 INSTANTIATE_TEST_SUITE_P(
     XpressLpParameterTest, LpParameterTest,
-    testing::Values(LpParameterTestParams(SolverType::kXpress,
-                                          /*supports_simplex=*/true,
-                                          /*supports_barrier=*/true,
-                                          /*supports_first_order=*/false,
-                                          /*supports_random_seed=*/false,
-                                          /*supports_presolve=*/false,
-                                          /*supports_cutoff=*/false,
-                                          /*supports_objective_limit=*/false,
-                                          /*supports_best_bound_limit=*/false,
-                                          /*reports_limits=*/false)));
+    testing::Values(LpParameterTestParams(
+        SolverType::kXpress,
+        /*supports_simplex=*/true,
+        /*supports_barrier=*/true,
+        /*supports_first_order=*/true,
+        /*supports_random_seed=*/false,  // Xpress supports this but it does not
```

Comment:
I found that increasing the problem size from n = 5 to n = 20 ([here](https://github.com/google/or-tools/blob/93dafb79a0261b5211f4eb17437aeebd7f45e543/ortools/math_opt/solver_tests/lp_parameter_tests.cc#L132)) gave enough variability in Xpress.  

However, as a user of Xpress, I'm not convinced that it's OK for Xpress to always give out the same solution for small problems (n = 5). I even ran the code with `for (int seed = 10; seed < 20000; seed += 10)` (1999 different seeds), and only got one solution. In comparison, Highs reaches its max of 120 different solutions with the first 703 seeds (and 100 solutions with the first 203 seeds)

Reply:
I checked with the developers and that seems intended behavior. The random seed in Xpress does not have as big an impact in Xpress as it may have in other solvers.